connectivity/drivers/ble/FEATURE_BLE/COMPONENT_BlueNRG_MS/mbed_lib.json

```diff
@@ -1,20 +1,20 @@
 {
     "name": "bluenrg_ms",
     "config": {
-        "SPI_MOSI":  "ARDUINO_UNO_SPI_MOSI",
-        "SPI_MISO":  "ARDUINO_UNO_SPI_MISO",
-        "SPI_nCS":   "ARDUINO_UNO_A1",
-        "SPI_RESET": "ARDUINO_UNO_D7",
-        "SPI_IRQ":   "ARDUINO_UNO_A0",
-        "SPI_SCK":   "ARDUINO_UNO_D3",
+        "SPI_MOSI":  "D11",
```

Comment:
Good point, hardcoding this to D11, how to overwrite it as this is acomponent to be used by other targets. It was chosen  to be usable with arduino uno. A question is, shouldn't this be a specific config and available to be overriden by a target?

"SPI_MOSI": "MBED_CONF_BLUENRG_MOSI", etc ?

Reply:
This `mbed_lib.json` is the very place to generate MBED_CONF_..., so we shouldn't reference MBED_CONF_... in `mbed_lib.json`. The field `target_overrides` is the exact place for overriding pins per target.

In my opinion, providing defaults still make sense. This is convenient for targets supporting a common form factor (e.g. ARDUINO_UNO) so they don't need any overrides.